common/reviews/api/node-core-library.api.md

```diff
@@ -602,6 +608,11 @@ export interface IWaitForExitWithStringOptions extends IWaitForExitOptions {
     encoding: BufferEncoding;
 }
 
+// @public (undocumented)
+export interface IWeightedIterable {
```

Comment:
This type doesn't describe an iterable, so this should probably be called something like `IWeighted`.